test/cases/image_tests.sh

```diff
@@ -5,6 +5,13 @@ set -euo pipefail
 source /etc/os-release
 ARCH=$(uname -m)
 
+# Set WORKSPACE to current dir if unset
+WORKSPACE=${WORKSPACE:=$(pwd)}
+
+# Set variables for tests to pass on s390x
+BUILD_ID=${BUILD_ID:='s390x'}
+DISTRO_CODE=${BUILD_ID:='EL8'}
+
```

Comment:
Since the testing is not running on regular Jenkins slave, these variables are unset and so I have to set them to something, I don't know what is appropriate though.

Reply:
See Jenkinsfile for DISTRO_CODE.  BUILD_ID I believe is the ID of the Jenkins build job. You can replace that with the beaker job id if that's available.